cg/services/illumina/backup/backup_service.py

```diff
@@ -278,37 +286,76 @@ def retrieve_archived_file(self, archived_file: Path, run_dir: Path) -> None:
             file_path=str(archived_file), target_path=str(retrieved_file)
         )
 
+    @staticmethod
+    def parse_dsmc_output_sequencing_path(dsmc_output: list[str]) -> list[PdcSequencingFile]:
+        """Parses the DSMC command output to extract validated sequencing paths."""
+        validated_responses = []
+        for line in dsmc_output:
+            if is_pdc_sequencing_path(line):
+                parts = line.split()
+
+                fileDateTime: datetime = convert_string_to_datetime_object(
+                    f"{parts[DsmcOutput.DATE_COLUMN_INDEX]} {parts[DsmcOutput.TIME_COLUMN_INDEX]}"
+                )
+
+                query_response: PdcSequencingFile = PdcSequencingFile(
+                    dateTime=fileDateTime,
+                    path=Path(parts[DsmcOutput.PATH_COLUMN_INDEX]),
+                )
+                validated_responses.append(query_response)
+
+        return validated_responses
+
     @classmethod
-    def get_archived_sequencing_run_path(cls, dsmc_output: list[str]) -> Path | None:
+    def get_latest_archived_sequencing_run_path(cls, dsmc_output: list[str]) -> Path | None:
```

Comment:
If we cannot find an archived run we should end with an error and try not to return None.

Reply:
I do agree. We should address this, but we will need to change some logic to do so. It might be better to handle this in a new, fresh PR.